server/webapp/WEB-INF/rails.new/webpack/views/dashboard/pipeline_header_widget.js.msx

```diff
@@ -45,9 +45,9 @@ const PipelineHeaderWidget = {
     const analyticsIcons = [];
     if (pipeline.canAdminister && !pipeline.isDefinedInConfigRepo()) {
       const settingsPath = vnode.attrs.isQuickEditPageEnabled ? pipeline.quickEditPath : pipeline.settingsPath;
-      settingsButton     = (<a class={`edit_config`} href={settingsPath}/>);
+      settingsButton     = (<a class={`edit_config`} arial-label="Edit Pipeline Configuration" title={'Edit Pipeline Configuration'} href={settingsPath}/>);
```

Comment:
Typo?

Reply:
@naveenbhaskar in case it's hard to see what the typo is, it probably should be `aria-label` rather than `arial-label`. Same goes for the rest of the comments here.